### ISSUE TITLE
Update Monolog to 2.x

### ISSUE THREAD

**Issue by tldev-de** [Geotab/mygeotab-php#4]
Hi everyone,

I want to integrate the geotab sdk into a laravel application. Due to the dependency `monolog` locked to version 1.23 this is not possible:

```
  Problem 1
    - Root composer.json requires geotab/mygeotab-php ^1.1 -> satisfiable by geotab/mygeotab-php[1.1.0].
    - geotab/mygeotab-php 1.1.0 requires monolog/monolog ^1.23 -> found monolog/monolog[1.23.0, ..., 1.x-dev] but the package is fixed to 2.1.1 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
```

I wonder whether this repo is still active/supported or not. For integrating the SDK into modern php applications, the upgrade to monolog 2 is a very important step! I saw there is a pull request, which is open since April (see https://github.com/Geotab/mygeotab-php/pull/3). Are there any plans to accept and merge this pull request? Are there any reasons, why this pull request is not merged yet?

Best regards
Tobi

**Comment by rictd** [comment]
+1 please merge this PR

**Comment by JoshuaKissoon** [comment]
+1, need this please, thank you!

**Comment by colonelchlorine** [comment]
Fixed in recent commit 342795c2058602bf9c4d67922e1b7e17bdd1084f
